packages/waas/src/intents/base.ts

```diff
@@ -7,7 +8,7 @@ import { Session } from '../session'
 export type Intent<T> = Omit<RawIntent, 'data' | 'signatures'> & { data: T }
 export type SignedIntent<T> = Omit<RawIntent, 'data'> & { data: T }
 
-const VERSION = '1.0.0'
+const VERSION = `1 (Web ${PACKAGE_VERSION})`
```

Comment:
what is the "1" in front here for..?

Reply:
it's the intent version, perhaps would be better to make it explicit...

Reply:
its fine, we can leave it for now, thanks for clarifying